scene.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"github.com/veandco/go-sdl2/sdl"
+	"log"
+	"time"
+)
+
+type scene struct {
```

Comment:
Is it possible in Go to create a structure field like `renderableObjects [ ]*renderableObjectInterface` that contains different structures that implement this interface? And the iterate over this field calling common methods? 
When I tried to do so a lot of errors popped up :(

Reply:
Something like this?
```go
type renderableObjectInterface interface {
	fooMethod() string
}
type renderableObjects []renderableObjectInterface

type implementation struct {
}

func (i implementation) fooMethod() string {
	return "bar"
}

func foo() {
	bar := renderableObjects{}
	for _, obj := range bar {
		result := obj.fooMethod()
		fmt.Println(result)
	}
}
```

You can't use `*` when the type is an interface:
```go
renderableObjects [ ]renderableObjectInterface
```

Reply:
Yes, like this, thanks!